services/src/contexts/mod.rs

```diff
@@ -41,6 +42,8 @@ pub trait Context: 'static + Send + Sync + Clone {
     type LayerDB: LayerDb;
     type QueryContext: QueryContext;
     type ExecutionContext: ExecutionContext;
+    type TaskContext: TaskContext;
+    type TaskDb: TaskDb<Self::TaskContext>;
```

Comment:
it's not really a DB because we never want to persist the tasks right? Maybe we should call it differently? TaskManager?

Reply:
[599847d](https://github.com/geo-engine/geoengine/pull/555/commits/599847d1fafd57a440769b4b041f8a3efec1134f)

Reply:
[0940ad4](https://github.com/geo-engine/geoengine/pull/555/commits/0940ad4d915a828e7ac6ef9608c7ff1b8f3e3116)